src/sass/styles.scss

```diff
@@ -87,12 +88,18 @@ $border-thin: 1px solid $color-mid-light !default;
 @import "selectable_main_table";
 @import "settings_page";
 @import "snapshots";
+@import "status_bar";
 @import "storage_detail_overview";
 @import "storage_pool_form";
 @import "storage_volume_form";
 @import "storage";
+@import "toast";
 @import "upper_controls_bar";
 
+body {
+  overflow: hidden;
+}
+
```

Comment:
This is probably not needed.

Reply:
this is needed for the animation of translating along the y-axis. During the animation it overflows the body which causes a scrollbar to appear. If we don't want this styling maybe we can get rid of the sliding animation and just have fade in/out?

Reply:
Ah right. I think sliding is the right animation type here. We could probably have a wrapper for the NotificationList and apply the overflow hidden to that wrapper only. But then again, I am not sure if this is even a better approach. Maybe keep as is -- but better add a comment to this CSS. Because currently, it is unclear why this overflow hidden on body is needed.